app/src/main/java/openfoodfacts/github/scrachx/openfood/features/analyticsusage/AnalyticsUsageDialogFragment.kt

```diff
@@ -45,10 +48,15 @@ class AnalyticsUsageDialogFragment : BottomSheetDialogFragment() {
         return binding.root
     }
 
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
     private fun saveAnalyticsReportingPref(value: Boolean) {
         sharedPreferences
-                .edit()
-                .putBoolean(getString(R.string.pref_analytics_reporting_key), value)
-                .apply()
+            .edit()
+            .putBoolean(getString(R.string.pref_analytics_reporting_key), value)
+            .apply()
```

Comment:
```suggestion
            .edit { putBoolean(getString(R.string.pref_analytics_reporting_key), value) }
```

And import the required extension.